### PR TITLE
Replace TSS Coils with Titanium-Platinum Ones

### PR DESCRIPTION
**Pull request by Steelux8**
- Created a new material, TPV-Alloy, made from Titanium, Platinum and Vanadium (3/3/1);
- Replaced the Tungstensteel Coil Block with a TPV one, to make it properly included in EV while adding a Platinum sink in that tier;

The actual recipe for this new coil is not defined here, I'll have to change it in coremod. Additionally, the coil is still known internally as Tungstensteel, which can be changed later - I just wanted to make sure it works fine in-game.

The coil progression is not consistent with the tiers: LV has Cupronickel, MV has Kanthal, HV has Nichrome, and the next three are Tungstensteel, HSS-G and HSS-S. Since the TSS coils need 2 stacks of ingots for one EBF, it's easy to say that those three coil tiers are IV, and EV has none. That is the change this PR is for, to replace TSS coils with ones made of an alloy of Titanium, Platinum and Vanadium (the IRL alloy has Aluminium instead of Platinum and different proportions, but I wanted this to also be a plat sink while in EV).

* https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/363

### CODE REVIEW

**Comment by boubou19** [comment]
is it really necessary? for me the platinum sink is too much.

**Comment by Steelux8** [comment]
> is it really necessary? for me the platinum sink is too much.

It's 55 Platinum per EBF with these new coils. I found it to be a good amount, it that too much? If yes, the ratio of metals could be changed.

**Comment by chochem** [comment]
EV is when you can make the platline (the platinum part) and where the quests for it are too. If you do that then 55 dusts is nothing. Sounds like a pretty good incentive for me - without being a hard wall.

**Comment by Steelux8** [comment]
I simplified the metal inputs (Titanium/Platinum/Vanadium), from 6/6/2 to 3/3/1 since it's the same thing.

**Comment by Connor-Colenso** [comment]
Lets discuss this more after a stable release.

**Comment by Dream-Master** [comment]
> Lets discuss this more after a stable release.

Stable is out so discussion can start.

**Comment by Dream-Master** [comment]
> > Lets discuss this more after a stable release.
> 
> Stable is out so discussion can start.

I am Late EV now run a few cycles Platinum trough my chemical reactor and i see no issue about Plantinum.
And i am want to update my EBF to#. TSS rn so i can check if this is blance. TSS need much ebf time (HV 4 amp (EV) EBF with Nichrome coils)

File: src/main/java/gregtech/common/GT_Proxy.java [hunk 1]
```diff
@@ -2207,112 +2204,6 @@ public void onPlayerTickEventServer(TickEvent.PlayerTickEvent aEvent) {
                             tCount += tStack.stackSize * 64 / Math.max(1, tStack.getMaxStackSize());
                         }
                         if (this.mInventoryUnification) {
-
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/common/GT_Proxy.java:2210]: I don't understand what this code segment getting removed has to do with coils?

File: build.gradle [hunk 2]
```diff
@@ -693,6 +693,9 @@ if (modrinthProjectId.size() != 0 && System.getenv("MODRINTH_TOKEN") != null) {
             addModrinthDep(qual[0], qual[1], parts[1])
         }
     }
+    if (usesMixins.toBoolean()) {
```
Comment (Connor-Colenso) [inline build.gradle:696]: ???

**Review by Connor-Colenso — CHANGES_REQUESTED** [review]
There seems to be a lot of code in here that isn't actually related to new coils?

File: build.gradle (hunk 2 above)
Comment (boubou19) [reply build.gradle:696]: this is just the buildscript udpate, nothing to worry too much about

Comment (Dream-Master) [reply build.gradle:696]: update buildscript

File: src/main/java/gregtech/common/GT_Proxy.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/gregtech/common/GT_Proxy.java:2210]: this are the beed conversion code was removed. Not sure why it appears in this pr

**Review by Connor-Colenso — APPROVED** [review]

**Pull request merged**